boards.txt

```diff
@@ -17897,6 +17897,179 @@ heltec_wireless_stick_lite.menu.EraseFlash.all.upload.erase_cmd=-e
+heltec_wireless_tracker.menu.UploadMode.default.upload.use_1200bps_touch=false
+heltec_wireless_tracker.menu.UploadMode.default.upload.wait_for_upload_port=false
+heltec_wireless_tracker.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+heltec_wireless_tracker.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+heltec_wireless_tracker.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+
+heltec_wireless_tracker.menu.CPUFreq.240=240MHz (WiFi)
+heltec_wireless_tracker.menu.CPUFreq.240.build.f_cpu=240000000L
+heltec_wireless_tracker.menu.CPUFreq.160=160MHz (WiFi)
+heltec_wireless_tracker.menu.CPUFreq.160.build.f_cpu=160000000L
+heltec_wireless_tracker.menu.CPUFreq.80=80MHz (WiFi)
+heltec_wireless_tracker.menu.CPUFreq.80.build.f_cpu=80000000L
+heltec_wireless_tracker.menu.CPUFreq.40=40MHz
+heltec_wireless_tracker.menu.CPUFreq.40.build.f_cpu=40000000L
+heltec_wireless_tracker.menu.CPUFreq.20=20MHz
+heltec_wireless_tracker.menu.CPUFreq.20.build.f_cpu=20000000L
+heltec_wireless_tracker.menu.CPUFreq.10=10MHz
+heltec_wireless_tracker.menu.CPUFreq.10.build.f_cpu=10000000L
+
+heltec_wireless_tracker.menu.UploadSpeed.921600=921600
+heltec_wireless_tracker.menu.UploadSpeed.921600.upload.speed=921600
+heltec_wireless_tracker.menu.UploadSpeed.115200=115200
+heltec_wireless_tracker.menu.UploadSpeed.115200.upload.speed=115200
+heltec_wireless_tracker.menu.UploadSpeed.256000.windows=256000
+heltec_wireless_tracker.menu.UploadSpeed.256000.upload.speed=256000
+heltec_wireless_tracker.menu.UploadSpeed.230400.windows.upload.speed=256000
+heltec_wireless_tracker.menu.UploadSpeed.230400=230400
+heltec_wireless_tracker.menu.UploadSpeed.230400.upload.speed=230400
+heltec_wireless_tracker.menu.UploadSpeed.460800.linux=460800
+heltec_wireless_tracker.menu.UploadSpeed.460800.macosx=460800
+heltec_wireless_tracker.menu.UploadSpeed.460800.upload.speed=460800
+heltec_wireless_tracker.menu.UploadSpeed.512000.windows=512000
+heltec_wireless_tracker.menu.UploadSpeed.512000.upload.speed=512000
+
+heltec_wireless_tracker.menu.DebugLevel.none=None
+heltec_wireless_tracker.menu.DebugLevel.none.build.code_debug=0
+heltec_wireless_tracker.menu.DebugLevel.error=Error
+heltec_wireless_tracker.menu.DebugLevel.error.build.code_debug=1
+heltec_wireless_tracker.menu.DebugLevel.warn=Warn
+heltec_wireless_tracker.menu.DebugLevel.warn.build.code_debug=2
+heltec_wireless_tracker.menu.DebugLevel.info=Info
+heltec_wireless_tracker.menu.DebugLevel.info.build.code_debug=3
+heltec_wireless_tracker.menu.DebugLevel.debug=Debug
+heltec_wireless_tracker.menu.DebugLevel.debug.build.code_debug=4
+heltec_wireless_tracker.menu.DebugLevel.verbose=Verbose
+heltec_wireless_tracker.menu.DebugLevel.verbose.build.code_debug=5
+
+heltec_wireless_tracker.menu.EraseFlash.none=Disabled
+heltec_wireless_tracker.menu.EraseFlash.none.upload.erase_cmd=
+heltec_wireless_tracker.menu.EraseFlash.all=Enabled
+heltec_wireless_tracker.menu.EraseFlash.all.upload.erase_cmd=-e
+
```

Comment:
Can you please just move this part of EraseFlash at the end of board definition, to match other boards menus ordering. Thanks